monitor.py

```diff
@@ -250,7 +250,7 @@ def cluster_deploy_status(spy_link):
         if "sno" in spy_link:
             job_log_url = PROW_VIEW_URL + spy_link[8:] + '/artifacts/' + job_type + '/upi-install-powervs-sno/finished.json'
 
-        if version>=4.16:
+        if version>=4.16 and job_platform != "powervs":
```

Comment:
Please add code comments on why its not applicable for powervs. Applies to all new changes.

Reply:
Done.